libeos-updater-util/flatpak.c

```diff
@@ -873,10 +891,8 @@ read_flatpak_ref_actions_from_node (JsonNode      *node,
                                     EOS_UPDATER_ERROR,
                                     EOS_UPDATER_ERROR_UNKNOWN_ENTRY_IN_AUTOINSTALL_SPEC))
             {
-              g_warning ("%s while parsing %s. Skipping this action and it "
-                         "will not be reapplied later. System may be in an "
-                         "inconsistent state from this point forward.",
-                         local_error->message, filename);
+              g_autofree gchar *node_str = json_node_to_string (element_node);
+              g_ptr_array_add (skipped_action_entries, g_steal_pointer (&node_str));
```

Comment:
You can also just pass the return value of `json_node_to_string` directly to this function.